### PR TITLE
Thanksgiving Article Nov 2023

### PR DESCRIPTION
**Pull request by pagrubel**
Addresses:

* #1851 

By making a submission here, you agree to [BSSw.io content licensing terms](https://github.com/betterscientificsoftware/bssw.io/blob/main/TERMS.md).

You are welcome to submit an *idea* for an article or a completed article.

After your submission is recieved, it will be reviewed by the BSSw.io editorial staff and we will be in touch with you.

If BSSw.io editorial staff decide to move your submission forward, a [checklist](https://betterscientificsoftware.github.io/bssw.io/bssw_content_pr_checklist.html) of additional actions to take to complete the publication process may be pasted here.

### CODE REVIEW

**Comment by bernhold** [comment]
Hi @pagrubel I haven't been able to participate in the conversations about this.  It would be helpful to me to have a couple of examples from others before I try to write anything.  But with that help, I'm willing to try...

**Comment by bartlettroscoe** [comment]
It does not look like any text has been contributed yet.  Is there an example on what we would like to see?

Also, can we change the file name to add the year. like `EditorialThanks_2023.md`?  Otherwise, what would we name the file for next year?

**Comment by pagrubel** [comment]
I've added my story, please add yours and if you have any comments about how the scope or purpose of the article should be let me know.

**Comment by pagrubel** [comment]
@betterscientificsoftware/bssw-editorial-board Please add your story by Wednesday Nov 22

**Comment by rinkug** [comment]
@pagrubel : done (with first draft atleast)

**Comment by bartlettroscoe** [comment]
@betterscientificsoftware/bssw-editorial-board, I added my content.  I added too much but it will be easy to cut down once we decide on the desired length and focus of the article.

**Comment by haikudeb** [comment]
I have written my part but I'm not sure how or where to add it. I can see the pull request and I was able one time to get at the document with everyone's part but then I couldn't edit it and then it has become unfindable. Need some guidance. :(

**Comment by bernhold** [comment]
@haikudeb, if you want to add your contribution through the GitHub web interface, go to the `Files changed` tab at the top of the main PR page (i.e. this page), then select the ellipsis menu on the upper right side of the file diff (*below* the green `Review changes` button) and select `Edit`.

If you continue to have problems, past your text into a comment or mail it to me and I will add it.

Thanks

**Comment by haikudeb** [comment]
Thanks !! Sent from my iPhoneOn Nov 18, 2023, at 7:30 PM, David E. Bernholdt ***@***.***> wrote:﻿
@haikudeb, if you want to add your contribution through the GitHub web interface, go to the Files changed tab at the top of the main PR page (i.e. this page), then select the ellipsis menu on the upper right side of the file diff (below the green Review changes button) and select Edit.
If you continue to have problems, past your text into a comment or mail it to me and I will add it.
Thanks

**Comment by pagrubel** [comment]
@bernhold I think it is ready. If you want to add anything to liven up the intro, please take the liberty to do so.

File: Articles/Blog/EditorialThanks-2023.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+After this realization, I began applying many of the software engineering best practices to my CSE projects and realized significant improvements in productivity and the overall quality of the software I was writing.
+However, I was also horrified to realize that, prior to gaining this knowledge, I had created tens of thousands of lines of CSE software of insufficient quality and that I was now in a position to have to maintain.
+
+I got involved in the IDEAS project when it was first started in 2014 and worked with people from other DOE laboratories to write some of the precursor material for BSSw.io.
+Once the IDEAS project was consumed into the ECP project in 2017, the BSSw.io site was started, and it incorporated much of that material.
+Editing and contributed to BSSw.io was an opportunity for me to share some what I had learned with the rest of the broader CSE/HPC community and to learn from the community, from the other IDEAS-ECP members, and especially from the other BSSW.io editorial staff.
```
Comment (pagrubel) [inline Articles/Blog/EditorialThanks-2023.md:37]:
```suggestion
Editing and contributing to BSSw.io was an opportunity for me to share some what I had learned with the rest of the broader CSE/HPC community and to learn from the community, from the other IDEAS-ECP members, and especially from the other BSSW.io editorial staff.
```

**Comment by rinkug** [comment]
@pagrubel : This is a nice read. Thanks for leading this

**Pull request merged**